keyboards/le_chiffre/rules.mk

```diff
@@ -0,0 +1,22 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = atmel-dfu
+
+BOOTMAGIC_ENABLE = lite       # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes            # USB Nkey Rollover
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+TAP_DANCE_ENABLE = no
+ENCODER_ENABLE = yes
+OLED_DRIVER_ENABLE = no
```

Comment:
Template matching here. No logic changes.

```suggestion
# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
MIDI_ENABLE = no            # MIDI support
BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
AUDIO_ENABLE = no           # Audio output on port C6
FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
ENCODER_ENABLE = yes
```

Reply:
Where did Oled_Driver go? Board does not support hd44780, or audio on c6. Don't think I want to implement this as suggested.

Reply:
These changes align your file to the [QMK template](https://github.com/qmk/qmk_firmware/blob/adda5d77707ea8f73d3d717f88802031b3f3fe27/quantum/template/avr/rules.mk), plus the rule to enable rotary encoders. OLED Driver and Tap Dance are both disabled by default, so removing their rules from this file doesn't change anything.

Reply:
Actually, I've changed my mind about this. Go ahead and ignore this suggestion.